api/src/paths/project/{projectId}/survey/permits/list.ts

```diff
@@ -96,16 +97,19 @@ export function getSurveyPermits(): RequestHandler {
 
       await connection.open();
 
-      const surveyPermitsData = await connection.query(
-        getSurveyPermitsSQLStatement.text,
-        getSurveyPermitsSQLStatement.values
-      );
+      const response = await connection.query(getSurveyPermitsSQLStatement.text, getSurveyPermitsSQLStatement.values);
 
       await connection.commit();
 
-      const getSurveyPermitsData = (surveyPermitsData && surveyPermitsData.rows) || null;
+      const result = (response && response.rows) || null;
+
+      const getSurveyPermitsData = new GetPermitData(result);
+
+      if (!getSurveyPermitsData) {
+        return res.status(200).json(null);
```

Comment:
Are you wanting to send 200 null back? Or just an empty array? If you are sending null back maybe we should update openApiSchema to allow nullable?

Reply:
Thanks, good catch.   I made the permit section nullable.